eng/SourceBuild.props

```diff
@@ -38,7 +38,7 @@
       <InnerBuildArgs>$(InnerBuildArgs) --nodereuse false</InnerBuildArgs>
       <InnerBuildArgs>$(InnerBuildArgs) --warnAsError false</InnerBuildArgs>
       <InnerBuildArgs>$(InnerBuildArgs) --outputrid $(TargetRid)</InnerBuildArgs>
-      <InnerBuildArgs>$(InnerBuildArgs) /p:RuntimeOS=$(RuntimeOS)</InnerBuildArgs>
+      <InnerBuildArgs>$(InnerBuildArgs) /p:PackageOS=$(RuntimeOS) /p:ToolsOS=$(RuntimeOS)</InnerBuildArgs>
```

Comment:
should we rename the RuntimeOS property here too to avoid confusion?

or am I reading this right and it is overwritten by https://github.com/dotnet/installer/blob/7d35ae67bbe43ce9164074801b29e8e1bd942c27/src/SourceBuild/content/repo-projects/runtime.proj#L23

Reply:
Yes, source-build passes `RuntimeOS`, and we're using it to set `PackageOS` and `ToolsOS`.